css-pseudo-4/Overview.bs

```diff
@@ -1201,18 +1202,19 @@ Cascading and Per-Element Highlight Styles</h3>
         text-decoration-color: var(--decoration-color, red);
       }
     </pre>
-    A div's selection highlight would be a green background to the
-    selected content, with a 1px thick blue underline. The
-    ''--background-color'' custom property is not found on the
-    ''div::selection'' nor ''::selection'' so it is taken from the
-    ''::root'' custom property match. The ''--decoration-thickness''
-    property is inherited from ''::selection'' via the highlight
-    cascade and ''--decoration-color'' is found in
-    ''div::selection'' itself.
+    The universal selection uses the user-agent styling because it only defines
+    custom properties, with no properties that influence the appearance.
+    A div's selection highlight must be a green background to the  selected content,
+    with a 10px thick blue underline.
+    The ''--background-color'' and ''--decoration-thickness'' custom properties
+    are not found on the ''div::selection'' so come from the originating div element,
+    which itself inherits the custom properties from the root.
+    The ''--decoration-color'' is found in the ''div::selection'' itself,
+    so the color from the originating elemnent is not used.
```

Comment:
```suggestion
    However since ''--decoration-color'' is specified on the ''div::selection'' itself,
    so its value from the [=originating element=] is not used.
```